test/unit_tests/modules/util/dom-events-spec.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+var chai = require( 'chai' );
+var expect = chai.expect;
+var sinon = require( 'sinon' );
+var jsdom = require( 'mocha-jsdom' );
+var BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
+var ERROR_MESSAGES = require( BASE_JS_PATH + 'config/error-messages-config' );
+var domEvents = require( BASE_JS_PATH + 'modules/util/dom-events' );
+var input;
+var clicked;
+var message;
+
+describe( 'Dom Events bindEvent', function() {
+  jsdom();
+
+  beforeEach( function() {
+    input = document.createElement( 'input' );
+    input.type = 'checkbox';
+    input.value = 'test-bind-event';
+    clicked = false;
+  } );
+
+  it( 'should not update the var until the listener is added', function() {
+    input.click();
+    expect( clicked ).to.be.false;
```

Comment:
Is this test necessary? It seems like since `clicked` isn't tied to `input` there couldn't be a change in the codebase that would make this fail. I guess it could fail if `bindEvent` were added in the `beforeEach` block, but isn't that making the test test itself as opposed to the codebase API?